app/api/auth/logout/route.ts

```diff
@@ -0,0 +1,21 @@
+import { AxiosError } from 'axios';
+import { NextResponse } from 'next/server';
+import api from '@/api/serverSideAxiosConfig';
+
+export async function POST() {
+  try {
+    const gatewayResponse = await api.post('/auth/logout');
+    const res = new NextResponse(null, {
+      status: gatewayResponse.status,
+    });
+    res.cookies.delete('token');
+
+    return res;
```

Comment:
Do we need to redirect to the login/welcome page? Or is that handled elsewhere?

Reply:
Se esta haciendo client-side, voy a intentar cambiarlo

Reply:
No estaria pudiendo cambiarlo, intente hacer que el server de next devuelva un redirect pero el cliente no lo esta procesando. Deje el redirect del lado "cliente"